wordpress-mobile/WordPress-iOS
# Issue/10222 long look marks as read

Fixes #10222.

To test:
Push notifications tend to require manual testing. This particular PR can be validated by either triggering various notifications or by manually pushing an APNS payload.

The former requires you to configure Notification settings for a site you follow in the app. The latter involves the use of [Pusher](https://github.com/noodlewerk/NWPusher) or similar; instructions can be found [here](https://github.com/wordpress-mobile/WordPress-iOS/pull/9924). I'm happy to furnish you with sample payloads and/or the P12 file necessary to do this with fabricated payloads; feel free to contact me via Slack for assistance.

In any case, once you receive a _Comment_ push notification (i.e., this is the only type that's been merged to `develop`), you should be able to see that the remote notification has been marked as read. This can also be accomplished by manually debugging the content extension.
